app/views/home.html

```diff
@@ -1,12 +1,18 @@
-<h1>Pokédex Tracker</h1>
+<div class="home">
+  <div class="hero">
+    <img src="/pokeball.svg">
+    <h1>Pokédex Tracker</h1>
+  </div>
+
+  <div class="sub">
+    <h2>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out. Check out an example living dex <a class="link" href="https://pokedextracker.com/u/ashketchum10">here</a>!</h2>
 
-<div class="hero">
-  <img class="mockup" src="/mockup.png">
-  <p>A tool for tracking your <a href="http://bulbapedia.bulbagarden.net/wiki/Living_Pok%C3%A9dex" target="_blank" class="link">Living Dex</a> progress. Easily toggle between and track your captured Pokémon, find the locations of those left to be captured, and share a public link with others to see how you can help each other out.<br><br>This project is open source, and you can find the code on Github (<a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" class="link">API</a> &amp; <a href="https://github.com/robinjoseph08/pokedextracker.com" target="_blank" class="link">website</a>). Feel free to report issues, suggest features, or even submit a pull request!</p>
+    <p>This project is open source, and you can find the code on Github (<a href="https://github.com/robinjoseph08/api.pokedextracker.com" target="_blank" class="link">API</a> &amp; <a href="https://github.com/robinjoseph08/pokedextracker.com" target="_blank" class="link">website</a>). Feel free to report issues, suggest features, or even submit a pull request!</p>
```

Comment:
what do you think about flipping api and website since we wanna encourage clicking the website over the api since thats where the issues are?